extensions/eol-runtime-scanner/app.R

```diff
@@ -57,46 +77,43 @@ ui <- page_sidebar(
 
     checkboxInput(
       "use_r_cutoff",
-      "Match R versions…",
+      label = "R older than…",
       value = FALSE
     ),
     conditionalPanel(
       condition = "input.use_r_cutoff == true",
-      sliderTextInput(
-        "min_r_version",
+      selectizeInput(
+        "r_version_cutoff",
         label = NULL,
-        choices = "...",
-        pre = "≤ "
+        choices = NULL
       )
     ),
```

Comment:
I noticed the `select` boxes here didn't have labels. They are not shown which makes sense, but they aren't accessible due to this change since there is no content in the `<label>`. If we could supply a label and hide it with some CSS like the below that would be great.

```
.sr-only {
  position: absolute;
  width: 1px;
  height: 1px;
  padding: 0;
  margin: -1px;
  overflow: hidden;
  clip: rect(0, 0, 0, 0);
  white-space: nowrap;
  border-width: 0;
}
```